python/GafferArnoldUITest/InteractiveArnoldRenderPerformanceTest.py

```diff
@@ -0,0 +1,228 @@
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+##########################################################################
+
+import imath
+import math
+import os
+import time
+
+import IECore
+import IECoreScene
+
+import Gaffer
+import GafferArnold
+import GafferImage
+import GafferImageUI
+import GafferScene
+import GafferSceneTest
+import GafferTest
+import GafferUI
+import GafferUITest
+import GafferImageTest
+
+from Qt import QtCore
+
+class InteractiveArnoldRenderPerformanceTest( GafferUITest.TestCase ) :
+
+	def runInteractive( self, useUI, useBlur, resolution ):
+
+		script = Gaffer.ScriptNode()
+
+		script["Camera"] = GafferScene.Camera()
+		script["Camera"]["transform"]["translate"]["z"].setValue( 6 )
+
+		script["Sphere"] = GafferScene.Sphere( "Sphere" )
+		script["Sphere"]["radius"].setValue( 10 )
+
+		script["ImageShader"] = GafferArnold.ArnoldShader()
+		script["ImageShader"].loadShader( "image" )
+		script["ImageShader"]["parameters"]["filename"].setValue( os.path.dirname( __file__ ) + "/../GafferImageTest/images/GafferChecker.exr" )
+		script["ImageShader"]["parameters"]["sscale"].setValue( 16 )
+		script["ImageShader"]["parameters"]["tscale"].setValue( 16 )
+
+		script["ShaderAssignment"] = GafferScene.ShaderAssignment()
+		script["ShaderAssignment"]["in"].setInput( script["Sphere"]["out"] )
+		script["ShaderAssignment"]["shader"].setInput( script["ImageShader"]["out"] )
+
+		script["Group"] = GafferScene.Group()
+		script["Group"]["in"][0].setInput( script["Camera"]["out"] )
+		script["Group"]["in"][1].setInput( script["ShaderAssignment"]["out"] )
+
+		script["StandardOptions"] = GafferScene.StandardOptions()
+		script["StandardOptions"]["in"].setInput( script["Group"]["out"] )
+		script["StandardOptions"]["options"]["renderCamera"]["value"].setValue( '/group/camera' )
+		script["StandardOptions"]["options"]["renderCamera"]["enabled"].setValue( True )
+		script["StandardOptions"]["options"]["renderResolution"]["value"].setValue( imath.V2i( resolution, resolution ) )
+		script["StandardOptions"]["options"]["renderResolution"]["enabled"].setValue( True )
+
+		script["ArnoldOptions"] = GafferArnold.ArnoldOptions( "ArnoldOptions" )
+		script["ArnoldOptions"]["in"].setInput( script["StandardOptions"]["out"] )
+		# Make sure we leave some CPU available for Gaffer
+		script["ArnoldOptions"]["options"]["threads"]["value"].setValue( -1 )
+		script["ArnoldOptions"]["options"]["threads"]["enabled"].setValue( True )
+
+		script["Outputs"] = GafferScene.Outputs()
+		script["Outputs"].addOutput(
+            "beauty",
+            IECoreScene.Output(
+                "Interactive/Beauty",
+                "ieDisplay",
+                "rgba",
+                {
+                    "driverType" : "ImageDisplayDriver",
```

Comment:
Duplicate parameter - I presume the `ClientDisplayDriver` value is the one we actually want.

Reply:
Addressed.